src/main/java/com/gtnewhorizons/retrofuturabootstrap/api/ClassHeaderMetadata.java

```diff
@@ -26,8 +26,10 @@ public final class ClassHeaderMetadata implements FastClassAccessor {
     public final int thisClassIndex;
     public final int superClassIndex;
     public final int interfacesCount;
+    public final int @NotNull [] interfaceIndices;
     public final @NotNull String binaryThisName;
     public final @Nullable String binarySuperName;
+    public final String @NotNull [] binaryInterfaceNames;
```

Comment:
```suggestion
    public final @NotNull String @NotNull [] binaryInterfaceNames;
```
this means a non-null array of non-null strings